cmd/src/batch_common.go

```diff
@@ -55,6 +56,10 @@ func newBatchApplyFlags(flagSet *flag.FlagSet, cacheDir, tempDir string) *batchA
 		&caf.allowUnsupported, "allow-unsupported", false,
 		"Allow unsupported code hosts.",
 	)
+	flagSet.BoolVar(
+		&caf.allowIgnored, "no-ignore", false,
```

Comment:
We might want to make the name of the flag a bit more forceful (pun somewhat intended), just to reinforce that it probably shouldn't be used normally. `-force-override-ignore`, or something like that?

Reply:
Done!